File: Valuating/Resource/Scripts/dbserver/IndexRemarkServer.js
//指标评语操作类
DB.IndexRemarkServer = function() {
};
//保存指标评语 		参数(1) ALOS.Model.IndexRemarkType
DB.IndexRemarkServer.SaveRemark=function(type,success){
var array=new Array();
array[0]=type;

DB.CreateWebLog("IndexRemarkServer", "SaveRemark", array,success);
};
//根据指标ID获得指标评语 指标ID		参数(1) System.Int32
DB.IndexRemarkServer.RenderRemark=function(id,success){
var array=new Array();
array[0]=id;

DB.CreateWebLog("IndexRemarkServer", "RenderRemark", array,success);
};

File: DALSERVER/IndexRemarkServer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using ALOS.Expand;
using ALOS.Model;

namespace ALOS.DALSERVER
{
    /// <summary>
    /// 指标评语操作类
    /// </summary>
    [APIServer("指标评语操作类")]
    public class IndexRemarkServer : Help<IndexRemarkType>
    {
        /// <summary>
        /// 保存指标评语
        /// </summary>
        /// <param name="type"></param>
        /// <returns></returns>
        [APIServer("保存指标评语")]
        public IndexRemarkType SaveRemark(IndexRemarkType type)
        {
            return base.Save(type);
        }

       

        /// <summary>
        /// 根据指标ID获得指标评语
        /// </summary>
        /// <param name="id"></param>
        /// <returns></returns>
        [APIServer("根据指标ID获得指标评语","指标ID")]
        public IndexRemarkType RenderRemark(int id)
        {
            return base.Reader("zid", id)??new IndexRemarkType(){ZId = id};
        }
    }
}
